Process.yml

```diff
@@ -954,3 +961,25 @@ $graph:
     loadListing:
       type: LoadListingEnum?
       jsonldPredicate: "cwl:loadListing"
+
+
+- type: record
+  name: TimeLimit
+  extends: ProcessRequirement
+  doc: |
+    Set an upper limit on the execution time of a process.
+    A process whose execution duration exceeds the time limit may be
+    preemptively terminated and considered failed.  May also be
+    used by batch systems to make scheduling decisions.
```

Comment:
From the call today, we need to clarify that `TimeLimit` when applied at the `Workflow` level does flow to all steps and sub-steps but doesn't set a maximum run time for the collective `Workflow` all together.

Was suggested that we renamed `TimeLimit` to `ToolTimeLimit` to also make this clearer.

Reply:
Yup, I think that's why it would be good to point out that setting this at workflow level will propagate to tools, e.g something like this :) 
```
    If ToolTimeLimit is set on a workflow level, it propagates down to
    individual processes, in line with requirement inheritance rules. 
    ToolTimeLimit is not used to limit the execution time of a Workflow.
```